### PR TITLE
Configure and run Prettier, pass ESLint

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
+    ecmaVersion: 2019,
+    sourceType: "module",
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: ["airbnb-base"],
+  plugins: ["jest", "svelte3"],
+  rules: {
```
Comment (hamilton) [inline .eslintrc.js:12]: does this need the same check as https://github.com/graph-paper-org/graph-paper/pull/40/files#diff-e4403a877d80de653400d88d85e4801aR37?

Comment (openjck) [reply .eslintrc.js:12]: Yes, although that's not related to Prettier. I'll open another PR for getting the template to pass ESLint tests.

**Comment by openjck** [comment]
Updated

**Comment by openjck** [comment]
Updated this branch to also pass ESLint. It was easier than doing it on another branch.

**Pull request merged**